Update picture-elements.markdown: add "name" option to state-badge (#42723)
home-assistant/home-assistant.github.io: source/_dashboards/picture-elements.markdown

  description: '[Position and style the element](#how-to-use-the-style-object) using CSS.'
  type: map
  default: "position: absolute, transform: translate(-50%, -50%)"
name:
  required: false

>   description: Text label shown below a state badge (entity name is used if not defined). Set to null to hide.

Reviewer: ```suggestion
  description: An optional alternative name displayed below the state badge. Defaults to the entity name if not provided. Set to null to hide.
```

  type: string
title:
  required: false

>   description: State badge tooltip (entity name is used if not defined). Set to null to hide.

Reviewer: ```suggestion
  description: State badge tooltip. Defaults to the entity name if not provided. Set to null to hide.
```
